data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+
+
+
+
+ship "Refraktos"
+	plural "Refraktos"
+	sprite "ship/avgi refraktos"
+	thumbnail "thumbnail/avgi refraktos"
+	swizzle 0
+	attributes
+		category "Medium Warship"
+		licenses
+			"Avgi Military"
+		"shields" 3600
+		"hull" 8400
+		"required crew" 12
+		"bunks" 24
+		"mass" 180
+		"drag" 3.6
+		"heat dissipation" 2.4
+		"fuel capacity" 1800
+		"cargo space" 60
+		"outfit space" 510
+		"weapon capacity" 150
+		"engine capacity" 144
+		"anchor point" 1
+		weapon
+			"blast radius" 420
+			"shield damage" 880
+			"hull damage" 440
+			"hit force" 1320
+	outfits
+		"Phased Array Laser (Blue)" 4
+		"Phased Array Laser (Green)" 3
+
+		"Stellar Fusion Core"
+		"Photoelectric Scales"
+		"Ultracapacitor Bank" 2
+		"Diffuse Deflector"
+		"Optical Lasing Generator" 3
+		"Multispectral Scanner" 3
+		"Electrostatic Ramscoop"
+		"Thermal Sink Partition" 3
+		"Electromotive Rifle" 24
+		
+		"Z-360 Fusion Torch"
+		"XY-420 Radial Thrusters"
+		"Hyperdrive"
+
+	gun -24 -33
+		angle -90
+	gun 24 -33
+		angle 90
+	gun -24 -9
+		angle -90
+	gun 24 -9
+		angle 90
+	turret -18 -81 "Phased Array Laser (Blue)"
+	turret 18 -81 "Phased Array Laser (Blue)"
+	turret -18 -72 "Phased Array Laser (Blue)"
+	turret 18 -72 "Phased Array Laser (Blue)"
+	turret -21 -60 "Phased Array Laser (Green)"
+	turret 21 -60 "Phased Array Laser (Green)"
+	turret 0 54 "Phased Array Laser (Green)"
+
+	engine 0 150
+	"reverse engine" -9 162
+		angle -30
+	"reverse engine" 9 162
+		angle 30
+	"steering engine" -12 -108
+		angle 90
+		right
+	"steering engine" 12 -108
+		angle -90
+		left
+	"steering engine" -9 162
+		angle 90
+		left
+	"steering engine" 9 162
+		angle -90
+		right
+
+	leak "leak" 60 60
+	leak "flame" 60 90
+	leak "big leak" 30 60
+	explode "tiny explosion" 12
+	explode "small explosion" 18
+	explode "medium explosion" 18
+	explode "large explosion" 6
+	"final explode" "final explosion medium"
+
+	description "The Refraktos-class Laserstar is a relatively recent, post-Diaspora destroyer design with a single focus: directed energy weapons. While its armor and shielding systems are light for its size, its expansive radiators can support the thermal loads of a staggering array of laser weapons, both offensive and defensive, allowing it to destroy both missiles and small ships from a vast distance."
+	description "A long truss running the length of the ship is built to fit an impressively large mirror-cell fusion torch, giving the Refraktos the mobility needed to evade opponents until they are reduced to hot vapor."
+	description "Due to the anemic damage output of Avgi laser weapons compared to their kinetic weapons and missiles, the Refraktos is most often seen in a support role, providing antimissile coverage to entire fleets."
+
+
+# Heavy Warships
+
+ship "Atenuatia"
+	sprite "ship/avgi atenuatia"
+	thumbnail "thumbnail/avgi atenuatia"
+	swizzle 0
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Avgi Military"
```

Comment:
```suggestion
		licenses
			"Avgi Military"
		"cost" 14400000
```
Lower than the Navy Carrier's 15.2M cost, as the Atenuatia is only a carrier, not also a direct combatant, but still towards the high end of T1 heavy warship costs, because 24 bays is a lot, and this has a comparable HP total to high-end T1 heavy warships.